src/main/java/org/fcrepo/camel/FcrepoProducer.java

```diff
@@ -198,7 +175,8 @@ private String getContentType(final Exchange exchange) {
     private String getAccept(final Exchange exchange) {
         final Message in = exchange.getIn();
         final String fcrepoTransform = in.getHeader(FcrepoHeaders.FCREPO_TRANSFORM, String.class);
-        final String acceptHeader = getAcceptHeader(exchange);
+        final String acceptHeader = Optional.ofNullable(in.getHeader(Exchange.ACCEPT_CONTENT_TYPE, String.class))
+                                            .orElse(in.getHeader("Accept", String.class));
 
         if (!isBlank(endpoint.getTransform()) || !isBlank(fcrepoTransform)) {
```

Comment:
You could use [`Stream::of`](http://docs.oracle.com/javase/8/docs/api/java/util/stream/Stream.html#of-T...-) here with the values to check and then [`Stream::findFirst`](http://docs.oracle.com/javase/8/docs/api/java/util/stream/Stream.html#findFirst--) to get the first non-blank choice, with `Optional::orElse` for the case of `DEFAULT_CONTENT_TYPE`.